tests/Behat/MinkContext.php

```diff
@@ -174,7 +174,7 @@ protected function getLanguageSwitcherOverlay(): NodeElement {
    * @throws \Behat\Mink\Exception\ElementNotFoundException
    */
   protected function getBreadcrumb(): NodeElement {
-    $selector = 'nav.ecl-page-header-core__breadcrumb';
+    $selector = 'nav.ecl-page-header-standardised__breadcrumb';
```

Comment:
Let's get core or standardised from the site configuration, so we can run the same steps independently from the template we choose to use.